frontend/src/store/recipesStore.js

```diff
@@ -56,6 +61,25 @@ export const useRecipesStore = defineStore("recipes", {
 				}
 			}
 		},
+		async getFilteredRecipes() {
+			const searchUrl = `${this.baseUrl}/recipes/complexSearch`;
```

Comment:
From where have you defined this `baseUrl` ?

Reply:
It's in the state